### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by geoffmoss0**
…ken links

Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [X] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Changed wording to reflect that there is only one document and fixed broken links

### CODE REVIEW

**Review by mxmeinhold — DISMISSED** [review]
LGTM! :ship: :it:

**Comment by mxmeinhold** [comment]
Would you squash these commits into 1?

**Comment by geoffmoss0** [comment]
Squashed

**Review by mxmeinhold — APPROVED** [review]
LGTM! :ship: :it:

File: README.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # CSH Constitution
 The master branch's HEAD is the current legal constitution of CSH.
 
-Compiled version of both documents can be found below:
-#### [Articles](https://constitution.csh.rit.edu/articles.pdf)
-#### [By-Laws](https://constitution.csh.rit.edu/bylaws.pdf)
+A compiled version of the document can be found [here](constitution.csh.rit.edu).
```
Comment (mxmeinhold) [inline README.md:4]:
```suggestion
A compiled version of the document can be found [here](https://constitution.csh.rit.edu).
```
If the href is just `constitution.csh.rit.edu`, it gets interpreted as a local link (you can test it if you view the readme on your fork).

**Review by mxmeinhold — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (geoffmoss0) [reply README.md]: Fixed it

**Pull request merged**